lib/apple_sign_in/user_identity.rb

```diff
@@ -2,6 +2,7 @@
 
 module AppleSignIn
   class UserIdentity
+    class JWTValidationError < StandardError; end
```

Comment:
can we add some info to this custom error?
Like `fields`, `description` or something that let the dev know more about the validation error perse